ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/assertion/statementofmeans/StatementOfMeansAssert.java

```diff
@@ -0,0 +1,112 @@
+package uk.gov.hmcts.cmc.ccd.assertion.statementofmeans;
+
+import org.assertj.core.api.AbstractAssert;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCollectionElement;
+import uk.gov.hmcts.cmc.ccd.domain.statementofmeans.CCDBankAccount;
+import uk.gov.hmcts.cmc.ccd.domain.statementofmeans.CCDCourtOrder;
+import uk.gov.hmcts.cmc.ccd.domain.statementofmeans.CCDDebt;
+import uk.gov.hmcts.cmc.ccd.domain.statementofmeans.CCDExpense;
+import uk.gov.hmcts.cmc.ccd.domain.statementofmeans.CCDIncome;
+import uk.gov.hmcts.cmc.ccd.domain.statementofmeans.CCDStatementOfMeans;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.BankAccount;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.CourtOrder;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.Debt;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.Expense;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.Income;
+import uk.gov.hmcts.cmc.domain.models.statementofmeans.StatementOfMeans;
+
+import java.util.List;
+import java.util.Objects;
+
+import static uk.gov.hmcts.cmc.ccd.assertion.Assertions.assertThat;
+
+public class StatementOfMeansAssert extends AbstractAssert<StatementOfMeansAssert, StatementOfMeans> {
+
+    public StatementOfMeansAssert(StatementOfMeans actual) {
+        super(actual, StatementOfMeansAssert.class);
+    }
+
+    public StatementOfMeansAssert isEqualTo(CCDStatementOfMeans ccdStatementOfMeans) {
+        isNotNull();
```

Comment:
Shouldn't we check it in constructor then?

Reply:
nope, it's the object passed in this method to compare. constructor does not have this info.